### PR TITLE
[WFLY-13798]:Add a Galleon layer for embedded broker messaging.

### PR DESCRIPTION
**Pull request by ehsavoie**
* Adding the new layer `embedded-activemq`.
 * Adding a new feature group `embedded-activemq` to contain the
   configuration.

Jira: https://issues.redhat.com/browse/WFLY-13798
Proposal: https://github.com/wildfly/wildfly-proposals/pull/473

Supersedes https://github.com/wildfly/wildfly/pull/15277

### CODE REVIEW

File: ee-feature-pack/galleon-common/src/main/resources/feature_groups/embedded-activemq.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="embedded-activemq" xmlns="urn:jboss:galleon:feature-group:1.0">
```
Comment (yersan) [inline ee-feature-pack/galleon-common/src/main/resources/feature_groups/embedded-activemq.xml:2]: For ee-9 module, we are also generating a configuration that includes the embedded messaging broker. Together with this PR, you should also adjust the ee-9 module to reuse this feature group spec and avoid re-defining it there.

See: https://github.com/wildfly/wildfly/blob/8958a48418159da5b0e3fce03aeb9f211342f2b1/ee-9/galleon-content/src/main/resources/packages/docs.examples.configs/pm/wildfly/tasks.xml#L23-L26

**Review by yersan — CHANGES_REQUESTED** [review]

**Review by yersan — APPROVED** [review]
LGTM

**Review by jfdenise — APPROVED** [review]

**Comment by ehsavoie** [comment]
@bstansberry I've also added the remote naming part as we discussed on https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers/topic/Remote.20naming.20layer.3F

**Review by bstansberry — COMMENTED** [review]
@ehsavoie This LGTM, but the ts/int/basic test of the layer is still using test-feature-pack and its remote-naming layer. I don't think that affects the results CI is showing, but we should clean that up. I did that in #15878, which is just this plus a cleanup commit.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**